### PR TITLE
Added migrations to change id to uuid.

### LINKED ISSUE #172: Change all id fields to uuids
@sudeeptarlekar _a

- Take care of data reset on staging

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
This fixes #172 

@vipulnsward _a Please review

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class Category < ApplicationRecord
+  default_scope -> { order("created_at ASC") }
```
Comment (vipulnsward) [inline app/models/category.rb]: Add this to ApplicationRecord instead.

File: db/migrate/20180708102747_change_column_types_to_uuids.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class ChangeColumnTypesToUuids < ActiveRecord::Migration[5.2]
+  def up
+    execute("ALTER TABLE articles ALTER COLUMN category_id SET DATA TYPE UUID USING (uuid(lpad(replace(text(category_id),'-',''), 32, '0')))");
+    execute("ALTER TABLE articles ALTER COLUMN organization_id SET DATA TYPE UUID USING (uuid(lpad(replace(text(organization_id),'-',''), 32, '0')))");
+
+    execute("ALTER TABLE article_urls ALTER COLUMN article_id SET DATA TYPE UUID USING (uuid(lpad(replace(text(article_id),'-',''), 32, '0')))");
+    execute("ALTER TABLE article_urls ALTER COLUMN url_id SET DATA TYPE UUID USING (uuid(lpad(replace(text(url_id),'-',''), 32, '0')))");
+
+    execute("ALTER TABLE urls ALTER COLUMN organization_id SET DATA TYPE UUID USING (uuid(lpad(replace(text(organization_id),'-',''), 32, '0')))");
```
Comment (vipulnsward) [inline db/migrate/20180708102747_change_column_types_to_uuids.rb]: No need for this. We will just flush all data on staging.

**Comment by sudeeptarlekar** [comment]
assigning to @sudeeptarlekar _a as test cases needs a fix

**Comment by sudeeptarlekar** [comment]
@vipulnsward _a PR is ready for review

File: app/javascript/Client/Request/Helpers.elm [hunk 3]
```diff
@@ -82,7 +82,7 @@ httpGet apiKey context url queryParams decoder =
     let
         -- NOTE: This should not be hardcoded or needed at all. It is right now hardcoded since this is the only entry we have in db
         tempBase =
-            "http://aceinvoice.com"
+            "http://aceinvoice.com/pricing"
```
Comment (breezyboa) [inline app/javascript/Client/Request/Helpers.elm]: @sudeeptarlekar http://aceinvoice.com was a temporary base url. pricing should not be appened here

**Comment by breezyboa** [comment]
@sudeeptarlekar _a
elm changes are done

**Comment by breezyboa** [comment]
@sudeeptarlekar _a
elm side chages are done

Note that only one category and one article are visible now rest is not

**Comment by sudeeptarlekar** [comment]
@vipulnsward _a all changes are done. Please review

**Comment by vipulnsward** [comment]
Please resolve conflicts

@sudeeptarlekar _a

**Comment by sudeeptarlekar** [comment]
@vipulnsward _a please review.

**Comment by vipulnsward** [comment]
Please check/setup data on staging

**Pull request merged**